Debugging.md

```diff
@@ -0,0 +1,9 @@
+#Debugging
+
+Like every other framework, Rhetos brings a new level of abstraction in the work of developers, but it also brings additional complications during debugging.
+
+A good thing is that the Rhetos server doesn't interpret DSL scripts "on the run", but on the basis of DSL scripts it generates the most common C# web service (source, dll and pdb) and a SQL database. When we need to debug a server side application, we can start Visual Studio and simply debug the Rhetos application like any other application that has been written in C#. Additionally, we sometimes use LinqPad to access the server object model, this is practical for smaller actions without installing and using Visual Studio.

```

Comment:
"the generated server application" instead of "a server side application" might be clearer. "Server-side" is used in a slightly different context.

Reply:
Changed to "the generated server application"